cms/src/main/java/uk/nhs/digital/ps/workflow/action/SearchableFlagAction.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.digital.ps.workflow.action;
```

Comment:
Can we have the packages structure by subject of the workflow operations rather than by technical aspect?:
```
workflow
  searchableflag
    SearchableFlagAction.java
    SearchableFlagTask.java
  searchabletaxonomy
    SearchableTaxonomyAction.java
    SearchableTaxonomyTask.java
```

Reply:
👍